py3status/modules/whoami.py

```diff
@@ -21,17 +19,26 @@ class Py3status:
     """
     """
     # available configuration parameters
-    cache_timeout = 1800
     format = '{username}'
 
+    class Meta:
+        deprecated = {
+            'remove': [
+                {
+                    'param': 'cache_timeout',
+                    'msg': 'obsolete parameter remove',
```

Comment:
```obsolete parameter``` would be good enogh no user sees this message currently it is more when parameters are renamed etc